covid/question_covid.py

```diff
@@ -33,46 +46,80 @@ def get_answer(self, text, question):
         start_scores, end_scores = self.MODEL(torch.tensor([input_ids]), token_type_ids=torch.tensor([token_type_ids]))
         all_tokens = self.TOKENIZER.convert_ids_to_tokens(input_ids)
         answer = ' '.join(all_tokens[torch.argmax(start_scores) : torch.argmax(end_scores)+1])
+        score = round(start_scores.max().item(), 2)
+        #response = self.nlp({'question': question, 'context': text})
+        #answer = response["answer"]
+        #score = response["score"]
 
-        return answer, start_scores.max(), end_scores.max()
+        return answer, score
 
-    def predict(self, question, data_text, index2paperID):
+    def predict(self, question):
 
-        query = self.TFIDF_VECTORIZER.transform([question])
+        query = self.TFIDF_VECTORIZER.transform([question + ' covid'])
```

Comment:
maybe this should be optional via an argument? Just thinking in the cases where you want to  ask generic questions about coronaviruses/sars rather than specifically to do with covid